### PR TITLE
Jenny parameter.optimization

### PR DESCRIPTION
**Pull request by Jennycocciardi**
Updated the ReadMe file to include process_radtags descriptive statistics and parameter optimization methods. This branch also includes some new scripts and added figures.

### CODE REVIEW

**Review by avahoffman — COMMENTED** [review]
What is this file? Can this be deleted?
https://github.com/avahoffman/urban-weed-genomics/blob/57dbf64609c501e93424b4f5564cfe873224848b/figures/process_radtags/EC/newfolder

File: bash/README.md [hunk 1]
```diff
@@ -248,28 +248,86 @@ barcode file:
...
+important to consider as cases where a single sample dominates the sublibary may
+indicate contamination.
+2. *The number of reads retained for each sample* can be an indicator of coverage.
+It is most likely a good idea to remove samples with a very low number of reads.
+Where you decide to place the cutoff for outliers is dependent on your dataset.
+For example, a threshold of 1 million reads is often used but this is not universal.
+3. *The proportion of reads retained for each sample* can also indicate low-quality
+samples and will give an idea of the variation in coverage across samples.
+
+The script `04-process_radtags_stats.R` was used to create plots for easily
```
Comment (avahoffman) [inline bash/README.md]: Where is this script? We should add it in :)

**Review by avahoffman — CHANGES_REQUESTED** [review]

**Comment by Jennycocciardi** [comment]
> What is this file? Can this be deleted? https://github.com/avahoffman/urban-weed-genomics/blob/57dbf64609c501e93424b4f5564cfe873224848b/figures/process_radtags/EC/newfolder

> Yes! Just deleted it.

File: bash/README.md (hunk 1 above)
Comment (Jennycocciardi) [reply bash/README.md]: Will add this in!

File: bash/README.md [hunk 2]
```diff
@@ -248,28 +248,86 @@ barcode file:
...
+In a new directory, make sure the files are organized by species. By
+default, files are sent to `~/scratch/demux/*sublibrary_name*`, but each
+file will need to be sent to `~/scratch/demux/*SPP*`. For example, the file
 “DS.MN.L01-DS.M.1.1.fq.gz” should be sent to the `~/scratch/demux/DS`
 directory.
 
 Note: this is not automated at this point but it would be nice to
 automate the file moving process so it’s not forgotten at this point.
 
+### Step 4c - Assess the raw, processed, and cleaned data
```
Comment (Jennycocciardi) [inline bash/README.md:267]: These aren't updating in the table of contents for some reason - I didn't touch the TOC, but hopefully I didn't mess anything up!

Comment (avahoffman) [reply bash/README.md:267]: It *should* update when merged! We should double check.

**Review by avahoffman — APPROVED** [review]
looks good!

**Pull request merged**